src/components/pages/InstructorHome/InstructorCalender.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Calendar } from 'antd';
+import './index.css';
+const InstrcutorCalender = () => {
+  function onPanelChange(value, mode) {
+    console.log(value, mode);
+  }
+  return (
+    <>
+      <section className="site-calendar-demo-card">
+        <div>
+          <h1>Your Calender</h1>
+          <div className="sub-items">
+            <h2>Subheader</h2>
+            <button>View Button</button>
+            <button>Add</button>
+          </div>
+        </div>
+        <Calendar fullscreen={false} onPanelChange={onPanelChange} />
```

Comment:
What does 'fullscreen={false}' do? Seems like it doesn't work like one would expect if the CSS was needed. Try looking into the Ant Design docs to see if there is a way to change the size by using props. Other than that, great job!